### DISCUSSION TITLE
Welcome to QtsApp Discussions!

### DISCUSSION THREAD

**Discussion by ShabbirHasan1** [TechfaneTechnologies/QtsApp#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AkilGoku** [comment]
Hey Doctor Joone ,  Kudos to all the great work you have been doing for the Indian stock market community, you are a great source of inspiration, looking forward to learn a lot from you :) 
Myself Akilan , a data scientist in retail domain , recently addicted to markets :)

**Reply by MooneDrJune** [reply]
Thanks For Your Kind and Inspiring Word's @AkilGoku 🙌
